### PR TITLE
Add podman init command

### PR DESCRIPTION
**Pull request by mheon**
As part of this, rework the number of workers used by various Podman tasks to match original behavior - need an explicit fallthrough in the switch statement for that block to work as expected.

Also, trivial change to Podman cleanup to work on initialized containers - we need to reset to a different state after cleaning up the OCI runtime.

Needs a test or two, bash completions, manpage

### CODE REVIEW

**Comment by baude** [comment]
needs: man page, integration tests, commands.md, and bash completions.

code lgtm

**Comment by TomSweeneyRedHat** [comment]
Any man page?
Change LGTM, test aren't too hip, don't know if it's related or not.

File: cmd/podman/init.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  podman init test1`,
+	}
+)
+
+func init() {
+	initCommand.Command = _initCommand
+	initCommand.SetHelpTemplate(HelpTemplate())
+	initCommand.SetUsageTemplate(UsageTemplate())
+	flags := initCommand.Flags()
+	flags.BoolVarP(&initCommand.All, "all", "a", false, "Stop all running containers")
```
[the remark this answers is not in the document]
Comment (mheon) [reply cmd/podman/init.go]: Oh, good point, there.

**Comment by mheon** [comment]
Tests added, manpage added, completions added, should be ready

File: cmd/podman/varlink/io.podman.varlink [hunk 2]
```diff
@@ -641,6 +641,13 @@ method StartContainer(name: string) -> (container: string)
 # ~~~
 method StopContainer(name: string, timeout: int) -> (container: string)
 
+# InitContainer initializes the given container. It accepts a container name or
+# ID, and will initialize the container matching that ID if possible, and error
+# if not. Containers can only be initialized when they are in the Created or
+# Exited states. Initialization prepares a container to be started, but does not
```
Comment (rhatdan) [inline cmd/podman/varlink/io.podman.varlink]: Can you add this description to the actual command --help.  I had no idea what init meant until I read this.
` Initialization prepares a container to be started, but does not start the container.`

Comment (rhatdan) [reply cmd/podman/varlink/io.podman.varlink]: Can you give me the user an idea why i would EVER want to do this?

Comment (mheon) [reply cmd/podman/varlink/io.podman.varlink]: Added some more context

Comment (edsantiago) [reply cmd/podman/varlink/io.podman.varlink]: Can you add that to the `--help` as well? Including perhaps a blurb about this being debug-only, not intended for regular users, etc etc? I can easily imagine a new user seeing this option and wasting much time trying to figure out how/when/why to use it. In fact I wonder if it shouldn't be marked hidden, and undocumented? Should this be user-visible at all?

Comment (mheon) [reply cmd/podman/varlink/io.podman.varlink]: I can see non-debug uses, but it's very much for advanced users - it's mostly for people who want the OCI spec created by the container so they can inspect and/or modify it.

Comment (edsantiago) [reply cmd/podman/varlink/io.podman.varlink]: I like it - brief and descriptive. Could you add that wording to the `--help` and the man page?

Comment (mheon) [reply cmd/podman/varlink/io.podman.varlink]: Done

**Comment by mheon** [comment]
Tests should hopefully be fixed

**Comment by mheon** [comment]
@baude I'm getting journald errors out of events that are blowing up the tests, can you take a look?

**Comment by mheon** [comment]
Alright, CI should actually be fixed this time

**Comment by jwhonce** [comment]
LGTM

**Comment by mheon** [comment]
All green
@baude @rhatdan PTAL and merge

**Pull request merged**